cache/cache_test.go

```diff
@@ -8,17 +8,44 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
+func TestExpirationWithJitter(t *testing.T) {
+	// Since expirationWithJitter use random, to multiple time to ensure
+	// result is within expected boundary
+	for n := 0; n < 1000; n++ {
+		expiration := 100 * time.Millisecond
+		jitter := 0 * time.Millisecond
```

Comment:
Shouldn't we test with a non-zero jitter though? This would effectively use the expiration time only.

Reply:
We do test with non-zero jitter just below. This one actually test that zero jitter disable the jitter behavior.

This test isn't needed in the loop (since it don't use randomization), do you want me to move it outside the loop ?

Reply:
Well, I've added a `TestExpirationWithoutJitter` test which is a better place to test with `jitter = 0` :)